### ISSUE TITLE
More than one programming language on one page?

### ISSUE THREAD

**Issue by KimStebel** [oreillymedia/thebe#69]
I guess this isn't possible at the moment? Ideally, it should be possible to use tmpnb to execute code in different languages on the same temporary notebook server.

**Comment by zischwartz** [comment]
Yeah, not possible at the moment. Or not with one instance of thebe anyway. It miiight be possible with multiple instances of thebe.

**Comment by KimStebel** [comment]
I tried to use multiple instances, but they seemed to get into each other's way.
